docker-compose.x86.darknet.cpu.yml

```diff
@@ -0,0 +1,13 @@
+version: '2.4'
+
+services:
+  ml_api:
+    image: thespaghettydetective/ml_api_x86_64_darknet_cpu
```

Comment:
Can you also add the relevant files for building these images? So far we have made sure to check in all code and script necessary to recreate the entire cluster from public docker images.

Reply:
Another thing to remember is we want, if possible, all docker images to be multi-architecture. Please refer to https://github.com/TheSpaghettiDetective/obico-server/blob/release/scripts/build_dockerfile_web_base.sh for how this can be done.